assets/sass/protocol/includes/_mixins.scss

```diff
@@ -2,8 +2,9 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@use "sass:math";
+/* stylelint-disable declaration-block-no-duplicate-properties */
```

Comment:
Ok, there are a lot of duplicate properties in this file. If we fix those, can this comment then be removed? 😅